frontend/packages/integration-tests-cypress/support/index.ts

```diff
@@ -82,8 +86,8 @@ after(() => {
 });
 
 export const checkErrors = () =>
-  cy.window().then((win) => {
-    assert.isTrue(!win.windowError, win.windowError);
+  cy.window().then(({ windowError }) => {
+    assert.isTrue(windowError && windowError.length === 0, windowError.toString());
```

Comment:
If `windowError` is undefined or null, won't this evaluate to false and tests will fail?

Reply:
Thanks for the catch! I've just changed the condition to check for the type instead